package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -253,3 +253,12 @@ function foreach_radio(uci, f)
 		end
 	end
 end
+
+function get_uptime()
+	local uptime_file = readfile("/proc/uptime")
+	if uptime_file == nil then
+		-- Something went wrong reading "/proc/uptime"
+		return nil
+	end
+	return tonumber(string.sub(uptime_file, 0, string.find(uptime_file, "%.") - 1))
```

Comment:
I think it would make sense to split at the space, not the dot. How about this?
```lua
return tonumber(uptime_file:match('^[^ ]+'))
```

Reply:
yup, works too